src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js

```diff
@@ -16,11 +16,15 @@ const filterTimeEntriesOnDate = (resourceDateProperty, date) => {
 };
 
 export const buildTimeEntriesFilter = (date, userId) => {
+  const ownerAndUserId = "ownerId=='" + userId + "'";
```

Comment:
It’s an even more pedantic comment than it was before but could _this_ be a template string?
```suggestion
  const ownerAndUserId = `ownerId==${userId}'`;
```

Reply:
yes it can committed.